src/ScaffoldInstallerPlugin.php

```diff
@@ -72,6 +72,130 @@ public static function getSubscribedEvents()
         ];
     }
 
+    /**
+     * Composer configuration advice: Use local copies of patch files.
+     *
+     * @return void
+     */
+    private function _checkComposerPatchesAreLocal()
```

Comment:
I don't think we need to prefix these with an underscore - they're private methods?

Reply:
I got a VSCode hint when I started moving the code to the Drainpipe repo branch, and I thought it was a coding standard in the Drainpipe project. It is probably just a misconfiguration on my IDE. I will remove the `_` from the checks I created.